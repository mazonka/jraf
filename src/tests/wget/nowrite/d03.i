command=jw data 0 1 /file_yes Kg=