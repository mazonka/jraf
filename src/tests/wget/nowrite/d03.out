JRAF_FAIL deny data
file_yes
